server.js

```diff
@@ -0,0 +1,34 @@
+var http = require('http');
+var url = require('url');
+
+
+var server = http.createServer(function(req, res) {
+  var path = url.parse(req.url).pathname;
+
+
+  if(path === '/time'){
+    var timeNow = new Date(Date.now()).toISOString();
+    res.writeHead(200, {'Content-Type': 'text/plain'});
```

Comment:
Since all three branches have same header, you could move this line to top of function.